types/cloudflare/index.d.ts

```diff
@@ -43,7 +43,7 @@ declare namespace Cloudflare {
     }
 
     interface DnsRecordWithPriority {
-        type: Exclude<RecordTypes, 'MX' | 'SRV' | 'URI'>;
+        type: Extract<RecordTypes, 'MX' | 'URI'>;
```

Comment:
I believe `SRV` needs to be included:
```
Required for MX, SRV and URI records; unused by other record types. Records with lower priorities are preferred
```

```diff
- type: Extract<RecordTypes, 'MX' | 'URI'>;
+ type: Extract<RecordTypes, 'MX' | 'SRV' | 'URI'>;
```

Reply:
If you look at the whole file contents, just below my changes, you will find the Definition of the SRV record, that a previous author added ;)

Reply:
Oops, didn’t click in my brain